eva-lib/src/main/java/uk/ac/ebi/eva/lib/configuration/DbCollectionsProperties.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2016-2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.lib.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@ConfigurationProperties(ignoreUnknownFields = false, prefix = "db.collections")
+@Component
+public class DbCollectionsProperties {
+
+    @Size(min = 1)
+    private String version;
+
+    @NotNull
+    private Files files;
+
+    @NotNull
+    private Variants variants;
```

Comment:
Isn't there a way to build this without extra classes, something like a nested namespace? Maybe this just shows how useless the extra level is... Should we drop the `name` suffix from the collection properties in the pipeline instead of adding it to the web services, @jmmut and @jorizci ?